Automattic/jetpack
# Like Button: add title attribute to like post iframe

Fixes #20069 

#### Changes proposed in this Pull Request:

* The like button which renders within iframe element was missing `title` attribute, issuing a WCAG notice when the site is tested against it. I've added a `title` attribute to comply with WCAG guidelines. Also, added an `aria-hidden` attribute to hide from the accessibility API.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Go to the site with the jetpack Like button enabled. 
- Run the post/page through a WCAG tester such as https://accessibleweb.com/website-accessibility-checker
- Make sure you don't see any notice related to the Frame title attributes.
